node/overseer/overseer-gen/tests/ui/ok-02.rs

```diff
@@ -0,0 +1,26 @@
+#![allow(dead_code)]
+
+use polkadot_procmacro_overseer_gen::overlord;
+
+
+struct X;
+
+#[derive(Default, Clone, Copy)]
+struct AwesomeSubSys;
+
+#[derive(Clone, AllSubsystemsGen)]
+#[overlord(Wrapper)]
+struct Overseer {
+	#[subsystem(X)]
+	sub0: AwesomeSubSys,
+}
+
+struct Spawner;
+
+fn main() {
+	let overseer = Overseer::<Spawner, SubSystems<X>>::builder()
```

Comment:
From an API perspective, this `SubSystems` appears from nowhere and is confusing

Reply:
This is the last instance and it is already flattened away. The gen.rs is to be removed.

Reply:
Yanked!